application.properties

```diff
@@ -1,7 +1,7 @@
 #Grails Metadata file
-#Mon Jul 21 16:57:34 EST 2014
+#Tue Aug 01 15:35:23 AEST 2017
 app.buildNumber=35
-app.grails.version=2.3.11
+app.grails.version=2.5.6
 app.name=ala-downloads
 app.servlet.version=2.5
```

Comment:
This line isn't necessary in Grails 2.5 - you can remove it to keep things tidy (also servlet version should be 3.0)